R/populate_multiple_folders.R

```diff
@@ -39,6 +44,15 @@ populate_multiple_folders <- function(outerdir.old = NULL,
                                       exe.file = "ss.exe",
                                       exe.only = FALSE,
                                       verbose = TRUE) {
+  lifecycle::deprecate_warn(
+    when = "1.46.2",
+    what = "populate_multiple_folders()",
+    details = paste(
+      "This function has been deprecated.",
+      "Please use r4ss::copy_SS_inputs() along with {purrr} or other tools."
```

Comment:
Thanks for the purrr and lapply examples. I suspect that nobody is using this function and it will be deleted before long, so I'm going to focus the example-writing on the documentation of copy_SS_inputs() where it will have a more lasting impact.